packages/core/src/login.ts

```diff
@@ -313,7 +314,7 @@ export function createAuthenticateFunction(options: CreateAuthFunctionOptions) {
         }
       };
 
-      if (isUsingSSL()) {
+      if (isUsingSSL() || isLocalhost()) {
```

Comment:
Do we have a way to implement an integration test to cover this behavior?

Reply:
We do have some units that cover OAuth, but don't have any integration to cover the whole process